test/State.mo

```diff
@@ -0,0 +1,196 @@
+import Debug "mo:base/Debug";
```

Comment:
🔥

Reply:
To be clear, I mean this test module is looking good. I'm not *that* impressed by `import Debug`...